problemtools/verifyproblem.py

```diff
@@ -947,8 +994,47 @@ def check(self, args):
                     self.warning('%s gets AC' % (desc))
             os.unlink(file_name)
 
+        self._verify_invalid_outputs()
+
         return self._check_res
 
+    def _verify_invalid_outputs(self):
+        """Check that output validators decline invalid answer files"""
+
+        val_timelim = self._problem.config.get('limits')['validation_time']
+        val_memlim = self._problem.config.get('limits')['validation_memory']
+        flags = self._problem.config.get('validator_flags').split()
+
+        for testcase in self._problem.testdata.get_all_testcases():
+            if not os.path.exists(testcase.infile):
+                # .in-file doesn't exist, already reported by testcase-check, so 'ignore' here
+                continue
+
+            wrong_answers = glob.glob(OutputValidators.WA_GLOB % testcase.ansfile)
+            if wrong_answers:
+                if self._problem.is_interactive:
+                    self.warning('Output validator check with wrong answers for interactive problems is currently not supported (skipping)')
+                    return
+
+                if self._problem.config.get('validation') == 'default':
+                    self.warning('Output validator check with wrong answers and default validator is unnecessary (skipping)')
```

Comment:
FWIW, I could see use cases for this, like declaratively saying that a trivial answer isn't the answer to any test case without having to manually check all the .ans files (which might even be hard to do with `diff` with floats involved). So if it's easy to remove this check it would be nice to do so.

Reply:
The problem here is that the default validator is not at self._validators (At least not in my tests), so every wrong output file is accepted.

Do you think that it's worthwhile to think about a solution here? With default validating, the trivial solution shouldn't be accepted because there is only one single solution normally.

Reply:
It's not super important, but if you can get it to work at the same time as factoring output validation out into a single function it might be worth doing. Otherwise skip it.

(The trivial solution might be accepted if it turns out that it is in fact correct, because test data was generated in a broken way, which IME happens surprisingly often. Of course, this is why one writes greedy/stupid solutions and checks that they get WA, but it might be theoretically useful to declare that on a per-test case level. So yeah, not a super strong use-case, but not completely pointless either.)

Reply:
Hm, `_actual_validators`?

Reply:
How could i miss that? Yes, that solves my problem. Works for default validation now.

Thank you very much!